seqr/models.py

```diff
@@ -459,6 +459,9 @@ class Individual(ModelWithGUID):
     def __unicode__(self):
         return self.individual_id.strip()
 
+    def __str__(self):
+        return self.individual_id.strip()
+
```

Comment:
Without adding `__str__()` to the `Individual` class, the matchmaker API will generate a bad `submission_guid`. The file `seqr/models.py` has been changed to Python 2 and 3 compatible. One the portion has to be changed was changed.